src/render/src/App.svelte

```diff
@@ -210,16 +282,50 @@ async function createCFG(params: Params): Promise<CFG> {
   }
 }
 
+let functionAndCFGMetadata: FunctionAndCFGMetadata | undefined;
+
+function updateMetadata(CFG: CFG, func?: SyntaxNode, language?: Language) {
+  // Update function metadata (GitHub only)
+  let name: string | undefined = undefined;
+  let lineCount: number | undefined = undefined;
+
+  if (func && language) {
+    name = extractFunctionName(func, language) ?? "<anonymous>";
+    lineCount = func.endPosition.row - func.startPosition.row + 1;
+  }
+
+  // Update CFG metadata
+  const nodeCount: number = CFG.graph.order;
+  const edgeCount: number = CFG.graph.size;
+  const cyclomaticComplexity: number = CFG.graph.size - nodeCount + 2; // (https://en.wikipedia.org/wiki/Cyclomatic_complexity)
+
+  return {
+    functionData: name && lineCount ? { name, lineCount, language } : undefined,
```

Comment:
Please make this a proper `if`, or _at least_ use parenthesis around the condition. Readability counts!